pkg/apis/postgres-operator.crunchydata.com/v1beta1/patroni_types.go

```diff
@@ -48,6 +52,21 @@ type PatroniSpec struct {
 	// - https://patroni.readthedocs.io/en/latest/kubernetes.html
 }
 
+type PatroniLogConfig struct {
+
+	// Patroni Log file size
+	// https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity
+	// +required
+	FileSize *string `json:"fileSize"`
```

Comment:
For the first question, ultimately, the value will be converted to the number of bytes, so I think that's probably ok. Maybe just add a note in the description that integers = number of bytes?

For the second question, yes more or less. That's probably the better way to go since the current implementation's defaulting behavior would be avoided altogether this way.

Reply:
Just to follow up on this, with the latest changes, the minimum value will equate to `25000000`, so hopefully that will lead to fewer configuration issues.